core-to-plc/src/Language/Plutus/CoreToPLC.hs

```diff
@@ -723,6 +723,6 @@ convExpr e = do
         -- ignore annotation
         GHC.Tick _ body -> convExpr body
         -- just go straight to the body, we don't care about the nominal types
-        GHC.Cast body _ -> convExpr body
+        GHC.Cast _ coerce -> unsupported $ "Unsupported: coercion" GHC.$+$ GHC.ppr coerce
```

Comment:
`coerce` is an often used function from `Data.Coerce`, so this line doesn't pass my personal `-Wall`. I'd make it `coe`, but this is nitpicking of course.

Reply:
TBH I think being very picky about name shadowing is silly anyway for things that are bound within a function.

Reply:
Well, that's what `-Wall` does. It's just that `PlutusPrelude` does not export `coerce`, but I like this function and use it often and others use it often.

Reply:
Sure, but shadowing *within the body of this match arm* is IMO not worth complaining about. Top-level shadowing sure.

Reply:
I worked with two guys who loved to shadow every commonly used name. It was super hard to read their code (`map map = ...`), so `-Wall` quickly started to be a bless for me.